Monarch Gene Mappings not found

I'm running `make mappings` to get the mappings, but I'm getting an error when downloading the `gene_mappings.tsv`. The folder `latest` is empty in the URL https://data.monarchinitiative.org/monarch-gene-mapping/index.html

```bash
mkdir -p mappings tmp
wget http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.tsv -O tmp/gene_mappings.sssom.tsv
--2023-11-15 11:24:57--  http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.tsv
Resolving data.monarchinitiative.org (data.monarchinitiative.org)... 35.208.191.193
Connecting to data.monarchinitiative.org (data.monarchinitiative.org)|35.208.191.193|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-11-15 11:24:57 ERROR 404: Not Found.

make[1]: *** [mappings/gene_mappings.sssom.tsv] Error 8
```

This is needed for #29

--- a/README.md
+++ b/README.md
@@ -72,3 +72,4 @@ _Note: If running on a Windows machine, replace `sh odk.sh` with `odk.bat` in th
 
 This project was made with the
 [mapping-commons-cookiecutter](https://github.com/mapping-commons/mapping-commons-cookiecutter).
+

--- a/monarch_mapping_commons.Makefile
+++ b/monarch_mapping_commons.Makefile
@@ -27,5 +27,5 @@ $(MAPPING_DIR)/mesh_chebi_biomappings.sssom.tsv:
 
 $(MAPPING_DIR)/gene_mappings.sssom.tsv:
 	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
-	wget http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.tsv -O $(TMP_DIR)/gene_mappings.sssom.tsv
+	wget http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.sssom.tsv -O $(TMP_DIR)/gene_mappings.sssom.tsv
 	sssom parse $(TMP_DIR)/gene_mappings.sssom.tsv -m $(METADATA_DIR)/gene_mappings.sssom.yml --prefix-map-mode merged -o $@

--- a/monarch_mapping_commons.Makefile
+++ b/monarch_mapping_commons.Makefile
@@ -28,7 +28,4 @@ $(MAPPING_DIR)/mesh_chebi_biomappings.sssom.tsv:
 $(MAPPING_DIR)/gene_mappings.sssom.tsv:
 	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
 	wget http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.tsv -O $(TMP_DIR)/gene_mappings.sssom.tsv
-	# see https://github.com/monarch-initiative/monarch-mapping-commons/issues/33
-	grep -v "<NA>" $(TMP_DIR)/gene_mappings.sssom.tsv > $@.tmp && mv $@.tmp $(TMP_DIR)/gene_mappings.sssom.tsv
-	grep -v ";" $(TMP_DIR)/gene_mappings.sssom.tsv > $@.tmp && mv $@.tmp $(TMP_DIR)/gene_mappings.sssom.tsv
 	sssom parse $(TMP_DIR)/gene_mappings.sssom.tsv -m $(METADATA_DIR)/gene_mappings.sssom.yml --prefix-map-mode merged -o $@

--- a/registry.yml
+++ b/registry.yml
@@ -23,5 +23,5 @@ mapping_set_references:
   - mapping_set_id: http://w3id.org/sssom/commons/monarch/gene_mappings.sssom.tsv
     mapping_set_group: gene_mappings
     local_name: gene_mappings.sssom.tsv
-    mirror_from: http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.tsv
+    mirror_from: http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.sssom.tsv
     mapping_set_confidence: "0.9"

--- a/registry.yml
+++ b/registry.yml
@@ -23,5 +23,5 @@ mapping_set_references:
   - mapping_set_id: http://w3id.org/sssom/commons/monarch/gene_mappings.sssom.tsv
     mapping_set_group: gene_mappings
     local_name: gene_mappings.sssom.tsv
-    mirror_from: http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.sssom.tsv
+    mirror_from: https://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.sssom.tsv
     mapping_set_confidence: "0.9"

